index.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const models = require('./models')
+const Op = require('sequelize').Op
+
+const app = express()
+
+app.get('/teams', async (request, response) => {
+    const teams = await models.teams.findAll()
+
+    response.send(teams)
+})
+
+app.get('/teams/:identifier', async (request, response) => {
+    const { identifier } = request.params
+
+    const matchingTeams = await models.teams.findAll({
+        where: {
+            [Op.or]: [{ id: identifier }, { abbreviation: identifier.toUpperCase() }]
+        }
+    })
+
+    if (matchingTeams.length) {
+        response.send(matchingTeams)
+    } else {
+        response.sendStatus(404)
+    }
+})
+
+app.use(bodyParser.json())
+
+app.post('/teams', async (request, response) => {
+    const { id, location, mascot, abbreviation, conference, division } = request.body
+
+    if (!id || !location || !mascot || !abbreviation || !conference || !division) {
```

Comment:
Here we are saying ID is a required field but it actually isn't and in fact we don't use it to create the team below because the database handles IDs for us. We should remove `id` from lines 33, 35, and 36 so we don't annoy users.